anvil/src/drawing.rs

```diff
@@ -66,6 +66,17 @@ pub fn draw_dnd_icon(
     }
 }
 
+pub fn draw_input_popup_surface(
+    surface: wl_surface::WlSurface,
+    location: impl Into<Point<i32, Logical>>,
+) -> SurfaceTree {
+    SurfaceTree {
+        surface,
+        position: location.into(),
+        z_index: 99, /* Cursor should always be on-top */
```

Comment:
This is not a cursor, isn't it? :thinking: 
Please update the comment and rather use 98 (so the cursor is the only thing on top of the input-popup).

Reply:
fixed(?)